### ISSUE TITLE
[Security] Users without any permissions can edit Localities

### ISSUE THREAD

**Issue by maxxxxxdlp** [specify/specify6#626]
**TL;DR: Users without View or Edit permission to tables can still make changes to the Locality table**

I created a user without any permissions (View, Edit, Remove) to any table
Though, the user has permission to view attachments
For attachments added to the `Locality` table, pressing the `i` button results in a window opening. That window has some information about the locality, information that should not be accessible to the user without table view privileges

![Screenshot from 2020-02-25 13-35-52](https://user-images.githubusercontent.com/40512816/75280833-ca2ec480-57d3-11ea-9444-d7433b947209.png)

Also, the `Show Locality` button is still enabled. Clicking that button loads a form without any data loaded:

![Screenshot from 2020-02-25 13-36-02](https://user-images.githubusercontent.com/40512816/75280836-cac75b00-57d3-11ea-8e95-090ed7bc2077.png)

Yet, the user is able to press the `Edit` button. That results not only in the user being able to edit all of the information but also in him being able to edit it and even save new data

Besides that, he has access to `GEOLocate`, `WordWind` and `Display in Google Earth` buttons
